src/table.ts

```diff
@@ -45,23 +46,28 @@ export type DropTableCallback = UpdateSchemaCallback;
 
 export type DeleteRowsCallback = CommitCallback;
 export type DeleteRowsResponse = CommitResponse;
-export type DeleteRowsOptions = CommitOptions;
+export interface DeleteRowsOptions extends CommitOptions {
+  transactionRequestOptions?: Pick<TxnRequestOptions, 'transactionTag'>;
+}
 
 export type InsertRowsCallback = CommitCallback;
 export type InsertRowsResponse = CommitResponse;
-export type InsertRowsOptions = CommitOptions;
+export type InsertRowsOptions = DeleteRowsOptions;
 
 export type ReplaceRowsCallback = CommitCallback;
 export type ReplaceRowsResponse = CommitResponse;
-export type ReplaceRowsOptions = CommitOptions;
+export type ReplaceRowsOptions = DeleteRowsOptions;
 
 export type UpdateRowsCallback = CommitCallback;
 export type UpdateRowsResponse = CommitResponse;
-export type UpdateRowsOptions = CommitOptions;
+export type UpdateRowsOptions = DeleteRowsOptions;
 
 export type UpsertRowsCallback = CommitCallback;
 export type UpsertRowsResponse = CommitResponse;
-export type UpsertRowsOptions = CommitOptions;
+export type UpsertRowsOptions = DeleteRowsOptions;
+
+type MutateRowsOptions = DeleteRowsOptions;
```

Comment:
Wouldn't it be more natural to define `MutateRowsOptions` as the extension of `CommitOptions` and then declare `DeleteRowsOptions = MutateRowsOptions` etc.?

So like this:

```typescript

interface MutateRowsOptions extends CommitOptions {
  transactionRequestOptions?: Pick<TxnRequestOptions, 'transactionTag'>;
}

export type DeleteRowsCallback = CommitCallback;
export type DeleteRowsResponse = CommitResponse;
export type DeleteRowsOptions = MutateRowsOptions;

export type InsertRowsCallback = CommitCallback;
export type InsertRowsResponse = CommitResponse;
export type InsertRowsOptions = MutateRowsOptions;

export type ReplaceRowsCallback = CommitCallback;
export type ReplaceRowsResponse = CommitResponse;
export type ReplaceRowsOptions = MutateRowsOptions;

export type UpdateRowsCallback = CommitCallback;
export type UpdateRowsResponse = CommitResponse;
export type UpdateRowsOptions = MutateRowsOptions;

export type UpsertRowsCallback = CommitCallback;
export type UpsertRowsResponse = CommitResponse;
export type UpsertRowsOptions = MutateRowsOptions;
```